app/src/main/java/org/simple/clinic/drugs/search/DrugFrequency.kt

```diff
@@ -58,4 +59,17 @@ sealed class DrugFrequency : Parcelable {
     @ToJson
     fun fromEnum(DrugFrequency: DrugFrequency?): String? = TypeAdapter.fromEnum(DrugFrequency)
   }
+
+
+  companion object {
+    fun fromMedicineFrequencyToDrugFrequency(value: MedicineFrequency?): DrugFrequency {
```

Comment:
Since this is a factory method on the `DrugFrequency` class, the `toDrugFrequency` suffix is redundant and can be removed.